pkgs/development/python-modules/glfw/default.nix

```diff
@@ -12,35 +7,29 @@ buildPythonPackage rec {
 
   disabled = pythonOlder "3.7";
 
-  src = fetchFromGitHub {
-    owner = "FlorianRhiem";
-    repo = "pyGLFW";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-3K+mDSz4ifVYkUvhd2XDPbhh6UCY4y54YqNLoAYDsP0=";
+  src = fetchPypi {
```

Comment:
why `fetchPypi` instead of `fetchFromGitHub`? building from source is always preferred

Reply:
I was under the impression it was the other way around. Reverted.